app/views/shared/_menu.html.erb

```diff
@@ -0,0 +1,9 @@
+<div style="text-align: right; padding-right: 3rem">
+
+  <nav>
+  <a href="/users/new">Sign up</a> |
+  <a href="/login">Log in</a> |
```

Comment:
One thing that would be good in the shared menu... once logged in we stop rendering login, signup etc.

We could provide a User.is_logged_in? method at the backend which is used to selectively render parts of the menu